core/templates/components/summary-tile/story-summary-tile.component.html

```diff
@@ -39,6 +39,7 @@
          [ngStyle]="{background: thumbnailBgColor}">
       <a [href]="storyLink">
         <img [src]="thumbnailUrl"
+             *ngIf="thumbnailUrl !== null"
```

Comment:
Minor nit: keep ngIf on the top and src below it. Makes it easier to read through logic in templates.

Reply:
Done